packages/buffetjs-core/src/components/DatePicker/index.js

```diff
@@ -81,6 +98,14 @@ class DatePicker extends React.PureComponent {
   showDatepicker = () => {
     this.setState({
       visible: true,
+      isFocused: true,
+    });
+  };
+
+  hideDatepicker = () => {
```

Comment:
The `hide` and `show` `Datepicker` instances are pretty much the same can you refactor them into a single one?